e2e_samples/dataset_versioning/insert_sql/main.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import pandas as pd
+import sys
+
+from keyvault_wrapper import KeyvaultWrapper
+from sql_wrapper import SqlWrapper
+from process import Process
+
+
+def read_csv(path: str) -> pd.DataFrame:
+    """
+    Read csv and modify datatype
+
+    :param path: path to csv
+    :type path: str
+    :return: dataframe with required columns
+    :rtype: pd.DataFrame
+    """
+    df = pd.read_csv(path)
+    df = df[["id", "loan_amnt", "annual_inc", "dti", "delinq_2yrs", "total_acc","total_pymnt", "issue_d", "earliest_cr_line", "loan_status"]]
+    df["issue_d"] = pd.to_datetime(df['issue_d'])
+    df['earliest_cr_line'] = pd.to_datetime(df['earliest_cr_line'])
+    return df
+
+
+def main(KeyvaultWrapper, SqlWrapper, Process, args_input):
+    """
+    main function to call argument parser to call the job chosen by the user
+
+    :param KeyvaultWrapper
+    :type Keyvaultwrapper: Keyvaultwrapper
+    :param SqlWrapper
+    :type SqlWrapper: SqlWrapper
+    :param Process
+    :type Process: Process
+    """
+    my_parser = argparse.ArgumentParser(description='Make sure to login')
```

Comment:
Do you want to provide ReadMe file to describe how to use this script with parameters ?

Reply:
Maybe I will write more on "help" and write readme to run ```main.py -h ```

Reply:
sounds great